allennlp/sanity_checks/task_checklists/textual_entailment_suite.py

```diff
@@ -0,0 +1,348 @@
+from typing import Optional, Tuple, Iterable
+from allennlp.sanity_checks.task_checklists.task_suite import TaskSuite
+from checklist.test_suite import TestSuite
+from checklist.test_types import MFT
+from checklist.perturb import Perturb
+import itertools
+import numpy as np
+from allennlp.sanity_checks.task_checklists import utils
+from overrides import overrides
+
+
+def wrap_apply_to_each(fn, both=False, *args, **kwargs):
```

Comment:
types?

Reply:
This looks like an internal function anyways, and should thus start with a `_`. Then I care less about types and docs too.